compiler/rustc_codegen_ssa/src/mir/rvalue.rs

```diff
@@ -332,7 +332,7 @@ impl<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> FunctionCx<'a, 'tcx, Bx> {
         scalar: abi::Scalar,
         backend_ty: Bx::Type,
     ) {
-        if matches!(self.cx.sess().opts.optimize, OptLevel::No | OptLevel::Less)
```

Comment:
@scottmcm FYI this is changing; I didn't see any discussion or experiments in https://github.com/rust-lang/rust/pull/109993 to support turning these off for `-Copt-level=1`; did you have some kind of compile time test that backed up having them off for that opt-level?